README.md

```diff
@@ -6,7 +6,9 @@ Confluent's .NET Client for Apache Kafka<sup>TM</sup>
 [![Chat on Slack](https://img.shields.io/badge/chat-on%20slack-7A5979.svg)](https://confluentcommunity.slack.com/messages/clients)
 
 **confluent-kafka-dotnet** is Confluent's .NET client for [Apache Kafka](http://kafka.apache.org/) and the
-[Confluent Platform](https://www.confluent.io/product/).
+[Confluent Platform](https://www.confluent.io/product/), which is a more complete distribution of
+Apache Kafka available in two editions - [Open Source](https://www.confluent.io/product/confluent-open-source/) and
```

Comment:
The wording in this PR is also used on our website. It resonated with me and is in fact what caused me to open this PR. I agree the words don't read as well (anywhere near as well). However a 'more complete version of Apache Kafka' is something i'm likely to click through to - I've arrived here so I already want Kafka - i'm definitely going to check out a more complete version of it. c.f. some 'platform' by some 'company'.